packages/suite-native-home/src/navigation/HomeStackNavigator.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+
+import { HomeDemoScreen } from '../screens/HomeDemoScreen';
+import { HomeScreen } from '../screens/HomeScreen';
+import { HomeStackParamList, HomeStackRoutes } from './routes';
+
+const HomeStack = createStackNavigator<HomeStackParamList>();
+
+export const HomeStackNavigator = () => (
+    <HomeStack.Navigator
+        initialRouteName={HomeStackRoutes.Home}
+        screenOptions={{ headerShown: false, gestureEnabled: true, gestureDirection: 'horizontal' }}
```

Comment:
Please move this config object to navigation package and just import it here since we will want this config same everywhere probably

Reply:
I've run into some TS issues with that so let's have a call about it tomorrow.

Reply:
[b7d1aa6](https://github.com/trezor/trezor-suite/pull/5617/commits/b7d1aa67768abc4a72c59c3d925329643a376396)